pkgs/development/tools/java/jprofiler/default.nix

```diff
@@ -0,0 +1,100 @@
+{ stdenv
+, lib
+, fetchurl
+, makeWrapper
+, makeDesktopItem
+, undmg
+, jdk11 }:
+
+let
+  inherit (stdenv.hostPlatform) system;
+  pname = "jprofiler";
+
+  # 11.1.4 is the last version which can be unpacked by undmg
+  # See: https://github.com/matthewbauer/undmg/issues/9
+  version = if stdenv.isLinux then "13.0.2" else "11.1.4";
```

Comment:
I haven't seen platform-dependent versions anywhere in Nixpkgs, especially major bumps. @SuperSandro2000 can we allow this?

Reply:
@azahi see example here: https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/libraries/science/math/mkl/default.nix

Reply:
OK, I guess it's alright then.